src/cli/fuzzilli_command.zig

```diff
@@ -21,7 +21,7 @@ pub const FuzzilliCommand = if (bun.Environment.enable_fuzzilli) struct {
         const reprl_script = @embedFile("../js/eval/fuzzilli-reprl.ts");
 
         // Create temp file for the script
-        var temp_dir = bun.FD.cwd().openDir("/tmp", .{}) catch {
```

Comment:
```error: ".stdDir()" is banned.
This PR increases the number of instances of this word from 42 to 43
Ban reason: Prefer bun.sys + bun.FD instead of std.fs.File. Zig hides 'errno' when Bun wants to match libuv
Or increase the limit by running \`bun ./test/internal/ban-words.test.ts --allow-increase\`
```

https://buildkite.com/bun/bun/builds/32192

Reply:
painge